docs/dev_guide/diagnostic_ref/dpct1040.rst

```diff
@@ -25,3 +25,78 @@ Suggestions to Fix
 * If the ``printf`` statement is only used from the host, do not change your code.
 * If the ``printf`` statement is only used from the device, use ``sycl::stream``
   instead of ``printf``.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    __host__ __device__ void hd() {
+      printf("Hello!\n");
+    }
+    
+    __global__ void k() {
+      hd();
+    }
+    
+    void foo() {
+      hd();
+      k<<<1, 1>>>();
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done